lib/screens/main_screen.dart

```diff
@@ -0,0 +1,169 @@
+import 'package:basic_widgets/screens/clip_rrect_screen.dart';
+import 'package:basic_widgets/screens/column_screen.dart';
+import 'package:basic_widgets/screens/container_screen.dart';
+import 'package:basic_widgets/screens/rich_text_screen.dart';
+import 'package:basic_widgets/screens/row_screen.dart';
+import 'package:basic_widgets/screens/sized_box_screen.dart';
+import 'package:basic_widgets/screens/stack_and_align_screen.dart';
+import 'package:basic_widgets/screens/wrap_screen.dart';
+import 'package:flutter/material.dart';
+import 'package:basic_widgets/strings.dart';
+
+class MainScreen extends StatelessWidget {
+  const MainScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: const Text(Strings.mainScreenTitle),
+        ),
+        body: getBody(context),
+        floatingActionButton: FloatingActionButton(
+          child: const Icon(Icons.add),
+          onPressed: () {
+            ScaffoldMessenger.of(context).showSnackBar(getSnackBar());
+          },
+        ),
+    );
+  }
+
+  SnackBar getSnackBar() {
```

Comment:
As an alternative, you can create a separate widget like `FloatingSnackBar` and use it here instead of this method.

Reply:
I create FloatingSnackBar class. And i extends it from SnackBar. Is it right or i must extends all classes for composition widgets from StateLessWidget?

Reply:
For FloatingSnackBar is ok because you need its specific behavior.